bin/kind-load

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eu
+
+if [ $# -eq 1 ]; then
+    cluster="${1:-}"
```

Comment:
what do you think supporting a default? `kind` itself uses `default` as the cluster name when it's not provided. personally i use `$USER`.